### PR TITLE
Support Version 2.0.0 changes for disregard-application #93

### PR DESCRIPTION
**Pull request by sunil-kumar-meena**
Fixed below issue

1. https://github.com/openBackhaul/TypeApprovalRegister/issues/93
2. https://github.com/openBackhaul/TypeApprovalRegister/issues/92

Note: Reviewer is @vanithavalluripalli9

### CODE REVIEW

File: server/service/IndividualServicesService.js [hunk 1]
```diff
@@ -150,7 +182,18 @@ exports.disregardApplication = function (body, user, originator, xCorrelator, tr
   });
 }
 
-
+/*
+* Delete profile by name from application-data/application-data.json
+*/
+const deleteProfileByName = (applicationData, applicationNameToDelete) => {
+  const applicationDataIndex = applicationData.findIndex(applicationDataItem => {
+     return applicationDataItem["application-name"] === String(applicationNameToDelete);
```
Comment (vanithavalluripalli9) [inline server/service/IndividualServicesService.js:190]: Pleas add check for release number as well

File: server/service/IndividualServicesService.js [hunk 2]
```diff
@@ -150,7 +182,18 @@ exports.disregardApplication = function (body, user, originator, xCorrelator, tr
   });
 }
 
-
+/*
+* Delete profile by name from application-data/application-data.json
+*/
+const deleteProfileByName = (applicationData, applicationNameToDelete) => {
```
Comment (vanithavalluripalli9) [inline server/service/IndividualServicesService.js:188]: can we change the function name as it is not profile we are deleting

**Review by vanithavalluripalli9 — CHANGES_REQUESTED** [review]

File: server/service/IndividualServicesService.js (hunk 1 above)
Comment (sunil-kumar-meena) [reply server/service/IndividualServicesService.js:190]: Code is checking both application name and release number at line 158

File: server/service/IndividualServicesService.js (hunk 2 above)
Comment (sunil-kumar-meena) [reply server/service/IndividualServicesService.js]: done

**Comment by sunil-kumar-meena** [comment]
Creating a new PR, so closing the old PR

**Review by vanithavalluripalli9 — CHANGES_REQUESTED** [review]
Kindly , move the functionality that includes read and write of application-data.json file and fetching applications list from application-data.json file to common file under ./individualServices

**Comment by sunil-kumar-meena** [comment]
> Kindly , move the functionality that includes read and write of application-data.json file and fetching applications list from application-data.json file to common file under ./individualServices

--this is done

**Review by vanithavalluripalli9 — DISMISSED** [review]
changes looks good

**Review by vanithavalluripalli9 — APPROVED** [review]

**Review by vanithavalluripalli9 — CHANGES_REQUESTED** [review]
Kindly , resolve the merge conflict of openapi.yaml file and include the changes for basic services as well

**Review by vanithavalluripalli9 — APPROVED** [review]

**Pull request merged**